emphasis_education_app/src/components/Chat/CreateChat.tsx

```diff
@@ -109,7 +106,7 @@ const CreateChat: React.FC<CreateChatProps> = ({ navigation }) => {
       ),
       headerBackTitle: 'Chats'
     })
-  }, [])
+  })
```

Comment:
this change causes the callback function to run after each render instead of just on mount. we are passing it params dynamically (through the state) so before, when it was being called, the function was not taking the most recent params.

it is possible that I will need to make these changes in the places where i use a button in the header to call a mutation.